cg/cli/workflow/mip/base.py

```diff
@@ -0,0 +1,117 @@
+import logging
+from pathlib import Path
+from typing import List
+
+import click
+
+from cg.apps.environ import environ_email
+from cg.exc import CgError
+from cg.meta.workflow.mip import MipAnalysisAPI
+from cg.cli.workflow.mip.options import (
+    OPTION_DRY,
+    OPTION_PANEL_BED,
+    EMAIL_OPTION,
+    PRIORITY_OPTION,
+    START_WITH_PROGRAM,
+    ARGUMENT_CASE_ID,
+)
+
+LOG = logging.getLogger(__name__)
+
+
+@click.command("config-case")
+@OPTION_DRY
+@OPTION_PANEL_BED
+@ARGUMENT_CASE_ID
+@click.pass_context
+def config_case(context: click.Context, case_id: str, panel_bed: str, dry_run: bool):
+    """Generate a config for the case_id"""
+    analysis_api: MipAnalysisAPI = context.obj["analysis_api"]
+    analysis_api.verify_case_id_in_statusdb(case_id)
+
+    panel_bed: str = analysis_api.resolve_panel_bed(panel_bed=panel_bed)
+
+    try:
+        config_data: dict = analysis_api.pedigree_config(case_id=case_id, panel_bed=panel_bed)
+    except CgError as error:
+        LOG.error(error.message)
+        raise click.Abort()
+    if dry_run:
+        print(config_data)
+        return
+    out_path: Path = analysis_api.write_pedigree_config(data=config_data, case_id=case_id)
+    LOG.info(f"Config file saved to {out_path}")
+
+
+@click.command()
+@OPTION_DRY
+@ARGUMENT_CASE_ID
+@click.pass_context
+def panel(context: click.Context, case_id: str, dry_run: bool):
+    """Write aggregated gene panel file"""
+
+    analysis_api: MipAnalysisAPI = context.obj["analysis_api"]
+    analysis_api.verify_case_id_in_statusdb(case_id=case_id)
+    bed_lines: List[str] = analysis_api.panel(case_id=case_id)
+    if dry_run:
+        for bed_line in bed_lines:
+            click.echo(bed_line)
+        return
+    analysis_api.write_panel(case_id, bed_lines)
+
+
+@click.command()
+@PRIORITY_OPTION
+@EMAIL_OPTION
+@START_WITH_PROGRAM
+@ARGUMENT_CASE_ID
+@OPTION_DRY
+@click.option("--mip-dry-run", "mip_dry_run", is_flag=True, help="Run MIP in dry-run mode")
+@click.option(
+    "--skip-evaluation",
+    "skip_evaluation",
+    is_flag=True,
+    help="Skip mip qccollect evaluation",
+)
+@click.pass_context
+def run(
+    context: click.Context,
+    case_id: str,
+    dry_run: bool = False,
+    email: str = None,
+    mip_dry_run: bool = False,
+    priority: str = None,
+    skip_evaluation: bool = False,
+    start_with: str = None,
+):
+    """Run the analysis for a case"""
+    analysis_api: MipAnalysisAPI = context.obj["analysis_api"]
+    analysis_api.verify_case_id_in_statusdb(case_id)
+    command_args = dict(
+        priority=priority or analysis_api.get_priority_for_case(case_id),
+        email=email or environ_email(),
+        dryrun=mip_dry_run,
```

Comment:
This looks weird 
```suggestion
        dry_run=mip_dry_run,
```

Reply:
This is how MIP sends it.... so dont wanna break anything